certbot/storage.py

```diff
@@ -759,7 +783,7 @@ def new_lineage(cls, lineagename, cert, privkey, chain, cli_config):
         """
 
         # Examine the configuration and find the new lineage's name
-        for i in (cli_config.renewal_configs_dir, cli_config.archive_dir,
+        for i in (cli_config.renewal_configs_dir, cli_config.default_archive_dir,
```

Comment:
No, it doesn't make it impossible - it was never possible to begin with. We could additionally add a command line flag to specify `archive_dir`, but otherwise, yes, that is how one would currently do it.